### PR TITLE
HAWQ-271 Remove netperf-2.4.3

### PR DESCRIPTION
**Pull request by radarwave**
1. Remove netperf tarball and Makefile.
2. Removed some debug output introduced by last merge.

### CODE REVIEW

File: tools/bin/gppylib/commands/base.py [hunk 1]
```diff
@@ -697,17 +697,10 @@ def __str__(self):
...
-            print "### DEBUG: EXECUTE name = %s" % ("SomeName" if self.name else "None")
-            print "### DEBUG: EXECUTE cmdStr = %s" % ("SomeCmdStr" if self.cmdStr else "None")
-            print "### DEBUG: EXECUTE context = %s" % ("SomeExecContext" if self.exec_context else "None")
-            print "### DEBUG: EXECUTE remoteHost = %s" % ("SomeRemoteHost" if self.remoteHost else "None")
             self.exec_context.execute(self)
         else:
             # simulate error
-            print "### DEBUG: CommandResult"
             self.results = CommandResult(1,'Fault Injection','Fault Injection' ,False,True)
             print self.results
```
Comment (huor) [inline tools/bin/gppylib/commands/base.py]: This also needs to be removed:
print self.results

**Comment by huor** [comment]
The fix looks good. +1

**Comment by linwen** [comment]
Looks good.

**Comment by xinzweb** [comment]
@hsyuan helped me work out a workaround on OSX.

If you are using OSX, this is a quick workaround we tried to get it working again:

    brew install postgresql
    sudo easy_install pip
    sudo pip --retries=50 --timeout=300 install pg8000 simplejson unittest2 pycrypto pygresql pyyaml lockfile paramiko psi
    sudo pip --retries=50 --timeout=300 install http://darcs.idyll.org/~t/projects/figleaf-0.6.1.tar.gz && \
    sudo pip --retries=50 --timeout=300 install http://sourceforge.net/projects/pychecker/files/pychecker/0.8.19/pychecker-0.8.19.tar.gz/download && \

Also, update the **greenplum_path.sh** to remove the **$GPHOME/ext/python/bin** so that default python can be used instead of the one come with hawq.

@radarwave since we no-longer depends on the python came with HAWQ, shall we just remote the **$GPHOME/ext/python**?

**Comment by cwelton** [comment]
All tests passing, two +1s.  What is preventing this PR from being resolved?

**Comment by radarwave** [comment]
Already merged, close the PR.

**Pull request closed without merging**